js-intro.js

```diff
@@ -19,43 +36,74 @@ var temp3 = 212
 
 var myNumbers1 = [3, 7, 0, 6, -9]
 
-
-
+const mult5 = (arrMult5) => {
+    let answerMult5 = []
+    for (let i=0; i<arrMult5.length; i++){
+    answerMult5.push(arrMult5[i] * 5)
+  }
+  return answerMult5
+}
+console.log(mult5(myNumbers1))
 
 // --------------------3) Create a function that multiplies each number in the array by 5 using map.
 // Use the test variable provided below. Expected outcome: [40, -35, 0, 30, 10]
 
 var myNumbers2 = [8, -7, 0, 6, 2]
 
-
-
+const mapMult5 = (array) => {
+  let answerMapMult5 = array.map(value => value * 5)
+  return answerMapMult5
```

Comment:
You can save yourself a line of code by returning your mapped array outright, rather than creating a new variable in order to turn around and immediately return it. Map is an accessor which doesn't permanently change to original array, so if you wanted to use those values later in your code you would need to save it to a new variable like you did. But for this purpose, you can save yourself some key strokes :) Also some small indentation issues here as well.